ingest/defaults/config.yaml

```diff
@@ -35,9 +35,7 @@ curate:
     geo-region: region
     geo-location: location
     isolate-collection-date: date
-    # This should really be date_released
-    # see <https://github.com/nextstrain/mpox/issues/104>
-    release-date: date_submitted
+    release-date: date_released
```

Comment:
Probably deserves a changelog entry?

Reply:
> I'm debating of keeping a copy of the release-date column as date_submitted in order to give people time to update their workflows.

Added in https://github.com/nextstrain/mpox/pull/319/commits/de47ccb51005a1aa6a40836e3a45ad442efb5292.